metabase/metabase
# Mantine Checkbox

Closes https://github.com/metabase/metabase/issues/32270

### Description

Adds styles and stories to the Mantine Checkbox and Checkbox group, based off the Figma file in the linked issue. I tried to follow the borders/spacing/etc as closely as possible - if there is some overkill in the code then I'm happy to simplify it.

### How to verify

Look at the Storybook for `Checkbox and Checkbox Groups` and see that the checkbox styling matches Figma. I'm happy to add more stories if necessary.

### Checklist

- [X] Stories have been added/updated to cover changes in this PR
